Review Form::Select Docs (#962)
hashicorp/design-system: website/docs/components/form/select/partials/code/how-to-use.md

</Hds::Form::Select::Field>
```

#### Custom control ID

> In case it's necessary to have custom ID for the control, instead of the one automatically generated by the component (eg. because it needs to be referenced in the code for other reasons), you just need to pass a `@id` argument to the "field":

Reviewer: ```suggestion
In cases where it’s necessary to have custom id attribute for the control, instead of the one automatically generated by the component (e.g., because it needs to be referenced somewhere else in the code), you just need to pass a `@id` argument to the "field":
```

</Hds::Form::Select::Field>
```

> #### HTML native attributes

Reviewer: ```suggestion
#### Native HTML attributes
```

This can be useful in case you want to add specific native behaviors to the field, that are not exposed directly by the component (eg. providing a `name` for the control, or adding `multiple` and `size` attributes to it)

> #### Events handling

Reviewer: ```suggestion
#### Event handling
```
